### PR TITLE
add unit in seconds for metrics resolution

### PR DESCRIPTION
**Pull request by jfcoz**
unit was missing in METRIC_RESOLUTION={{ openshift.hosted.metrics.resolution }}s

### CODE REVIEW

**Comment by detiber** [comment]
ok to test

**Comment by detiber** [comment]
aos-ci-test

**Comment by abutcher** [comment]
Should we add the unit to the value instead (and update the default in openshift_facts)?

```
openshift_hosted_metrics_resolution="10s"
```

**Comment by detiber** [comment]
@abutcher I agree, I think updating the default to have the unit and requiring the user to specify the unit would give more flexibility when setting the value.

File: roles/openshift_metrics/tasks/main.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@
   shell: >
    {{ openshift.common.client_binary }} process -f \
    /usr/share/openshift/examples/infrastructure-templates/{{ hawkular_type }}/metrics-deployer.yaml -v \
-    HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.{{ openshift.master.default_subdomain }},USE_PERSISTENT_STORAGE={{ hawkular_persistence }},METRIC_DURATION={{ openshift.hosted.metrics.duration }},METRIC_RESOLUTION={{ openshift.hosted.metrics.resolution }} \
+    HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.{{ openshift.master.default_subdomain }},USE_PERSISTENT_STORAGE={{ hawkular_persistence }},METRIC_DURATION={{ openshift.hosted.metrics.duration }},METRIC_RESOLUTION={{ openshift.hosted.metrics.resolution }}s \
```
Comment (abutcher) [inline roles/openshift_metrics/tasks/main.yaml]: Can remove the trailing `s` here.

File: playbooks/common/openshift-master/config.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@
       openshift_hosted_metrics_duration: "{{ lookup('oo_option', 'openshift_hosted_metrics_duration') | default(7) }}"
     when: openshift_hosted_metrics_duration is not defined
   - set_fact:
-      openshift_hosted_metrics_resolution: "{{ lookup('oo_option', 'openshift_hosted_metrics_resolution') | default(10) }}"
+      openshift_hosted_metrics_resolution: "{{ lookup('oo_option', 'openshift_hosted_metrics_resolution') | default('10s', true) }}"
```
Comment (abutcher) [inline playbooks/common/openshift-master/config.yml]: I don't think these set_facts are necessary? The default should be coming from [openshift_facts](https://github.com/openshift/openshift-ansible/blob/00eba039c9312fbd04cc05a8a890ef48f2311769/roles/openshift_facts/library/openshift_facts.py#L1725-L1764).

Comment (detiber) [inline playbooks/common/openshift-master/config.yml]: The set_fact here is for providing compatibility with bin/cluster and the way additional config can be specified.

Comment (abutcher) [inline playbooks/common/openshift-master/config.yml]: Got it.

**Comment by detiber** [comment]
aos-ci-test

**Comment by detiber** [comment]
bumping this back up. @jfcoz could you update the default in openshift_facts as well? Then I think this will be ready to go.

**Comment by detiber** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Pull request merged**